docs/docsite/rst/getting_started/index.rst

```diff
@@ -1,99 +1,94 @@
-.. _getting_started_index:
+Getting Started with Ansible in a Self-Contained Project Structure
+==================================================================
 
-############################
-Getting started with Ansible
-############################
+Introduction
+------------
+This guide will walk you through setting up a self-contained project structure for working with Ansible. The goal is to create a single directory that can easily be committed to a GitHub repository and used with tools like AWX and Navigator. Additionally, we will integrate Visual Studio Code (VSCode) and Ansible Lint for an improved development experience.
```

Comment:
I really don't think a getting started doc, should delve into setting up vscode or ansible-lint.  Any additional instructions like this should probably go into some additional guide.